widgets/AttributesTable/_GridMixin.js

```diff
@@ -419,6 +419,29 @@ define([
             }
             this.setToolbarButtons();
             topic.publish(this.attributesContainerID + '/tableUpdated', this);
+        },
+
+        clearSelectedGridRecords: function () {
```

Comment:
I think this should be `clearSelectedGridRows` to be consistent with dgrid's use of rows, not records in a dstore.